docs/source/sim_overview.rst

```diff
@@ -56,55 +54,28 @@ give you at least a basic exploratory idea of the performance of your review:
 	asreview plot myreview.h5
 	asreview plot DIR_WITH_MULTIPLE_SIMULATIONS
 
-	
+For an example of results of a simulation study, see the `Simulation Results <simulation-results.html>`__ page.
+
+
 Simulation options
 ------------------
-The options for simulating are shown on the `CLI <cli.html>`__ page. We will highlight some of the
-more often used options here.
+The options for simulating are shown on the `CLI <cli.html>`__ page. We will highlight some of the more often used options here. When no additional arguments are specified in the ``asreview simulate`` command, default settings are used.
+
+To make your simulations reproducible you can use the ``--seed`` and ``--init_seed`` options. 'Init
+seed' controls the starting set of papers to train the model on, while the 'seed' controls the
+random generator after initialization.
 
-To make your simulations reproducible you can use the ``--seed`` and ``--init_seed`` options. Init
-seed controls the starting set of papers to train the model on. While the normal seed controls the
-random generator after initialization. For initialization you can also use ``--prior_idx`` to set
-the papers to start the simulation with.
+By default, the model initializes with one relevant and one irrelevant record. If you want to initialize your model with a specific set of starting papers, you can use ``--prior_idx`` to select the indices of the papers you want to start the simulation with.
 
-To limit the size of your state file and the time to simulate, you can increase ``--n_instances``,
-which reduces the number of training iterations.
+The ``--n_instances`` argument controls the number of records that have to be labelled before the model is retrained again and is set at 1 by default. If you want to reduce the number of training iterations, for example to limit the size of your state file and the time to simulate, you can increase ``--n_instances``.
 
-Use a different classifier with the ``-m`` flag. Names for implemented models are listed on the
-`models <models.html>`__ page.
+You can select a classifier with the ``-m`` flag, which is set to be Naive Bayes by default. Names for implemented classifiers are listed on the :doc:`models <models.html>`page.
 
 Implemented query strategies are listed on the `query strategy <query_strategies.html>`__ page and
-is set with the ``-q`` option.
+can be set with the ``-q`` option.
 
-For feature feature extraction, supply the ``-e`` flag, with more details on the page for 
+For feature extraction, supply the ``-e`` flag. Default is TF-IDF, more details are on the page for
 `feature extraction <feature_extraction.html>`__.
 
-The last model that can be changed is the `balance strategy <balance_strategies.html>`__, and is
-changed with the ``-b`` flag.
-
```

Comment:
I also think this should be somewhere. Perhaps not on this page, that's true.

Reply:
The balance strategies are still in the document, they just moved a bit

Reply:
Yes, because I'm not sure it is anywhere else. We should at least link that page.